sympy/geometry/parabola.py

```diff
@@ -376,6 +376,28 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """Returns the line passing through the focus
```

Comment:
Note that [Wolfram ](https://mathworld.wolfram.com/LatusRectum.html)gives a different definition and we should probably stick with that: 

> The latus rectum of a conic section is the chord through a focus parallel to the conic section directrix (Coxeter 1969). "Latus rectum" is a compound of the Latin latus, meaning "side," and rectum, meaning "straight."